addon/-private/data-view/radar/radar.js

```diff
@@ -460,60 +532,46 @@ export default class Radar {
   }
 
   pageUp() {
+    if (this.renderAll) {
+      return; // All items rendered, no need to page up
+    }
+
     const {
       bufferSize,
-      renderedFirstItemIndex,
-      totalRendered
+      firstItemIndex,
+      totalComponents
     } = this;
 
-    if (renderedFirstItemIndex !== 0) {
-      const newFirstItemIndex = Math.max(renderedFirstItemIndex - totalRendered + bufferSize, 0);
+    if (firstItemIndex !== 0) {
+      const newFirstItemIndex = Math.max(firstItemIndex - totalComponents + bufferSize, 0);
       const offset = this.getOffsetForIndex(newFirstItemIndex);
 
-      this.scrollContainer.scrollTop = offset + this._scrollTopOffset;
+      this._scrollContainer.scrollTop = offset + this._scrollTopOffset;
       this.scheduleUpdate();
     }
   }
 
   pageDown() {
+    if (this.renderAll) {
+      return; // All items rendered, no need to page down
+    }
+
     const {
       bufferSize,
-      renderedLastItemIndex,
-      totalRendered,
+      lastItemIndex,
+      totalComponents,
       totalItems
     } = this;
 
-    if (renderedLastItemIndex !== totalItems - 1) {
-      const newFirstItemIndex = Math.min(renderedLastItemIndex + bufferSize + 1, totalItems - totalRendered);
+    if (lastItemIndex !== totalItems - 1) {
+      const newFirstItemIndex = Math.min(lastItemIndex + bufferSize + 1, totalItems - totalComponents);
       const offset = this.getOffsetForIndex(newFirstItemIndex);
 
-      this.scrollContainer.scrollTop = offset + this._scrollTopOffset;
+      this._scrollContainer.scrollTop = offset + this._scrollTopOffset;
       this.scheduleUpdate();
     }
   }
 
-  // If `renderAll` is true, render components for all items. We intercept this here because
-  // for all other behavior (action sending) we want to maintain the "correct" item indexes
-  get renderedFirstItemIndex() {
```

Comment:
These methods were originally convenience methods, but it felt like they were obfuscating what was actually going on here. We've relied too much on getters/setters in the past, it felt like it would be better to inline the logic since it's only really used in one place.